### PR TITLE
Ibm PowerVc events

### PR DESCRIPTION
**Pull request by Guddetisandeep**
As a starting point after adding these changes we are able to capture and parse the events from PowerVc in the ibm_power_vc provider

### CODE REVIEW

File: app/models/manageiq/providers/ibm_power_vc/cloud_manager/event_parser.rb
Comment (agrare) [inline app/models/manageiq/providers/ibm_power_vc/cloud_manager/event_parser.rb:1]: This looks like a full copy-paste from Openstack, if the code is the same then you don't actually need to define it here the openstack event_catcher/runner will call `event_hash = ManageIQ::Providers::Openstack::CloudManager::EventParser.event_to_hash(event, @cfg[:ems_id])`

Comment (Guddetisandeep) [reply app/models/manageiq/providers/ibm_power_vc/cloud_manager/event_parser.rb:1]: Okay, I will work on it and get the PR updated

**Review by jaywcarman — COMMENTED** [review]
Looking great @Guddetisandeep!

For PowerVC, only Ceilometer will work for the event stream, right? If so we should also update the cloud manager create form to remove the non-functional STF and AMQP options ([here](https://github.com/ManageIQ/manageiq-providers-ibm_power_vc/blob/master/app/models/manageiq/providers/ibm_power_vc/cloud_manager.rb#L223-L232)).

**Comment by Guddetisandeep** [comment]
@jaywcarman  I have confirmed with PowerVc team they said the PowerVC supports service for AMQP not STF

**Comment by jaywcarman** [comment]
> @jaywcarman I have confirmed with PowerVc team they said the PowerVC supports service for AMQP not STF

Thanks for checking @Guddetisandeep. Go ahead and remove the STF option from the `params_for_create`. We need to test AMQP if we're going to keep it as an option.

File: app/models/manageiq/providers/ibm_power_vc/cloud_manager/event_parser.rb [hunk 1]
```diff
@@ -1,10 +1,2 @@
 module ManageIQ::Providers::IbmPowerVc::CloudManager::EventParser
```
Comment (jaywcarman) [inline app/models/manageiq/providers/ibm_power_vc/cloud_manager/event_parser.rb:1]: Are you thinking we might need to override some of the parsing for PowerVC? If not I don't think there's a need to keep the empty module definition.

Comment (agrare) [reply app/models/manageiq/providers/ibm_power_vc/cloud_manager/event_parser.rb:1]: We shouldn't leave unused modules around, if/when you want to override event_parser then we can add this module

Comment (Guddetisandeep) [reply app/models/manageiq/providers/ibm_power_vc/cloud_manager/event_parser.rb]: Yeah, removed the `event_parser` and got it working

**Comment by agrare** [comment]
@Guddetisandeep diff looks good, could you clean up the commits though?  I'm not a fan of just squashing everything into 1 commit because I like to see the progression, but in cases where something was added and then removed in the same PR it isn't helpful for us in the future going back through the history.

Specifically these two commits look helpful:
https://github.com/ManageIQ/manageiq-providers-ibm_power_vc/pull/89/commits/2311bc3d05b5e4e45cf9df3370ead066c21b78ae
https://github.com/ManageIQ/manageiq-providers-ibm_power_vc/pull/89/commits/642df738fa2cd4d3a19ca9656c7794d895739c53

I think the 2x removed event parser ones can be dropped by simply amending the original commit which added it and the fixed indentation / remove duplicate code can be fixed up in the original commit.

**Comment by jaywcarman** [comment]
> We need to test AMQP if we're going to keep it as an option.

For the record, we decided to enable only Ceilometer for now. If/when we decide to test AMQP we'll enable it with another PR.

**Review by jaywcarman — APPROVED** [review]
LGTM

**Review by agrare — APPROVED** [review]
I went looking for the event_catcher worker additions but see this provider already has one.

**Pull request merged**